.github/workflows/notify-security-alerts.yml

```diff
@@ -0,0 +1,16 @@
+name: Send security alerts to slack
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+  workflow_dispatch:
+
+jobs:
+  send_security_alerts:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send security alert notifications to slack
+        uses: kunalnagarco/action-cve@v1.8.0
```

Comment:
It's a shame this action doesn't also send any code scanning or secret alerts as well 😞